### ISSUE TITLE
Source focuses on currently edited cell

### ISSUE THREAD

**Issue by michalanglart** [nokia/RED#148]
As described here: https://groups.google.com/forum/m/#!topic/robotframework-users/BtjBq_ZBtAE

the source editor scrolls to currently edited cell thus hiding the cells which follows in the same line.

**Comment by iysaleh** [comment]
Here's a gif of the issue (Screencapture is set to focus on the editor window--It's just modifying a really long line that is a wrapper around an external utility with arguments)

https://imgur.com/a/Dl0bP

RED version: 0.7.9.201706290901

**Comment by KrzysztofJozefowicz** [comment]
Can you provide more details as we have problems with reproduction?
What we need to know is:
OS,Eclipse version or RED version,sample testcase with step-by-step how to achieve it.

**Comment by iysaleh** [comment]
OS: Windows 10 X64
RED Version: Standalone RED Distribution that came in this zip: "RED_0.7.9.20170629090422-win32.win32.x86_64.zip"
I'm pretty sure Mac users have also reported this happening to them as well.

The only trick to recreating this is to make your keyword file a bit long and then it starts happening. Just modify the ${element_eleven} cell in the code block below and you should notice the editor autofocusing on it for some reason.

Sample Code (autofocus_bug_reproduction.robot):
```
*** Settings ***
Documentation     A Keyword library which causes the annoying autofocus behavior.
Library           BuiltIn
Library           OperatingSystem
Library           Collections
Library           Process

*** Keywords ***
Autofocus Keyword 1
    [Arguments]    ${arg_1}    ${arg_2}    ${arg_3}    ${arg_4}    ${arg_5}
    [Documentation]    This file just needs to get complex enough and then the auto-focus behavior kicks in. Here is the first keyword!
    ...    Another line of documentation
    ...    Yet another line of documentatino.
    ...    OMG, ANOTHER LINE!
    ${a_list}=    Create List    hello_world!
Autofocus Keyword 2
    [Arguments]    ${arg_1}    ${arg_2}    ${arg_3}    ${arg_4}    ${arg_5}
    [Documentation]    This file just needs to get complex enough and then the auto-focus behavior kicks in. Here is the first keyword!
    ...    Another line of documentation
    ...    Yet another line of documentatino.
    ...    OMG, ANOTHER LINE!
    ${a_list}=    Create List    hello_world!    
Autofocus Keyword 3
    [Arguments]    ${arg_1}    ${arg_2}    ${arg_3}    ${arg_4}    ${arg_5}
    [Documentation]    This file just needs to get complex enough and then the auto-focus behavior kicks in. Here is the first keyword!
    ...    Another line of documentation
    ...    Yet another line of documentatino.
    ...    OMG, ANOTHER LINE!
    ${a_list}=    Create List    hello_world!
Autofocus Keyword 4
    [Arguments]    ${arg_1}    ${arg_2}    ${arg_3}    ${arg_4}    ${arg_5}
    [Documentation]    This file just needs to get complex enough and then the auto-focus behavior kicks in. Here is the first keyword!
    ...    Another line of documentation
    ...    Yet another line of documentatino.
    ...    OMG, ANOTHER LINE!
    ${a_list}=    Create List    hello_world!
    Append to List    ${a_list}    ${element_one}    ${element_two}    ${element_three}    ${element_four}    ${element_five}    ${element_six}    ${element_seven}    ${element_eight}    ${element_nine}    ${element_ten}    ${element_eleven}    ${element_thirteen}    ${element_fourteen}    ${element_fifteen}
```
Please let me know if you still can't reproduce this.

**Comment by KrzysztofJozefowicz** [comment]
got it, thanks! It is reproducible on 0.8.0 and prerelease 0.8.1.  
Editing cell in long line gets aligned to right side of the screen so the slider moves. This is also related to number of lines or tokens in source editor - that's why we could not reproduce it before. We had similar case before ( #38 ),we will look into it although I doubt this would be fixed in upcoming release.

**Comment by iysaleh** [comment]
Wow, #38 seems like the exact same bug. I guess this has existed for over a year now at this point. That unfortunately means that the chance of seeing this actually get fixed is very low :( ... Good luck!

**Comment by KrzysztofJozefowicz** [comment]
#38 was fixed in 0.7.0, current one is new although may be related. Those kind of bugs are hard to spot in daily work or QA phase unless competent users file the report. Good that we can reproduce it on our site.

**Comment by KrzysztofJozefowicz** [comment]
released in 0.8.1
